src/app/core/navbar/navbar.component.css

```diff
@@ -118,3 +118,27 @@
   color: red;
   font-weight: bold;
 }
+
+.login-btn-container {
+  display: none;
+}
+.login-btn {
+  background-color: transparent;
+  border-width: 3px;
+  border-radius: 10px;
+  border-style: solid;
+  border-color: #fa8072;
+  padding: 5px !important;
+}
+.login-btn-mobile {
+  display: block;
+}
+
+@media (min-width: 768px) {
```

Comment:
@kishanmanoharan you should be able to use bootstrap breakpoints and classes for modifying the display property of an element. Try out the `d-block` and `d-none` classes and add breakpoints like `md`, `sm`.

Reply:
I've tried that, didn't work on my end. This solution feels like a last-resort. If we can find anything else that works, I can implement it.